src/main/java/org/keycloak/protocol/cas/CASLoginProtocol.java

```diff
@@ -120,7 +120,7 @@ public Response authenticated(AuthenticationSessionModel authSession, UserSessio
     }
 
     @Override
-    public Response sendError(AuthenticationSessionModel authSession, Error error) {
+    public Response sendError(AuthenticationSessionModel authSession, Error error, String str) {
```

Comment:
Third parameter should be named `errorMessage`, not `str`.

See: https://www.keycloak.org/docs-api/26.1.0/javadocs/org/keycloak/protocol/LoginProtocol.html